src/NamedTupleTools.jl

```diff
@@ -27,6 +28,8 @@ else
      export fieldtypes
 end
 
+struct NotPresent end
```

Comment:
Maybe a comment  explaining  what this is for

Reply:
added